bindings/py/nupic/bindings/check.py

```diff
@@ -22,36 +22,44 @@
 
 
 def checkImportBindingsInstalled():
+  """Check that nupic.bindings is installed (can be imported).
+
+  Returns the error message on failure and None if successful.
+  """
   try:
     import nupic.bindings
-  except ImportError:
-    return False
+  except ImportError as e:
+    return e.message
```

Comment:
In true Pythonic spirit, my preference would be to have this routines simply propagate the exceptions on failure instead of mapping the exceptions artificially to string or None.